test/blackbox/api/dds-pim/PubSubWriter.hpp

```diff
@@ -280,10 +280,11 @@ class PubSubWriter
     typedef typename type_support::type type;
 
     PubSubWriter(
-            const std::string& topic_name)
+            const std::string& topic_name,
+            eprosima::fastdds::dds::DomainParticipant* participant = nullptr)
         : participant_listener_(*this)
         , listener_(*this)
-        , participant_(nullptr)
+        , participant_(participant)
```

Comment:
No need to do it this way, we have [these methods](https://github.com/eProsima/Fast-DDS/blob/596780933e18001ded9ee20ccc2ab23ea349327b/test/blackbox/api/dds-pim/PubSubWriter.hpp#L1321-L1333) that allow setting the participant / writer properties

Reply:
Thank you, I haven't seen the method for setting the property to the participant